core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/blueprint/BlueprintV4Endpoint.java

```diff
@@ -38,7 +40,7 @@
     @Produces(MediaType.APPLICATION_JSON)
     @ApiOperation(value = BlueprintOpDescription.LIST_BY_WORKSPACE, produces = JSON, notes = BLUEPRINT_NOTES,
             nickname = "listBlueprintsByWorkspace")
-    BlueprintV4ViewResponses list(@PathParam("workspaceId") Long workspaceId);
+    BlueprintV4ViewResponses list(@PathParam("workspaceId") Long workspaceId, @DefaultValue("false") @QueryParam("withSdx") Boolean withSdx);
```

Comment:
withDatalake

Reply:
we use everywhere sdx (on public-facing), why should we use the datalake here?